custom_dc/build_repo.md

```diff
@@ -47,9 +47,10 @@ It will take several minutes to build.
 
 To run the container with the local SQLite database, start the Docker container as described below. 
 
-To run the container with a remote Cloud SQL database, see [Start the Docker container with Cloud data](build_repo.md#docker-data) for procedures.
+To run the container with a remote Cloud SQL database, see [Start the Docker container with Cloud data](/custom_dc/build_repo.html#docker-data) for procedures.
```

Comment:
Is this the right page to link to? This looks like it leads to the same page, and I don't see a #docker-data heading.